.babelrc

```diff
@@ -0,0 +1,13 @@
+{
+    "presets": [
+      [
+        "babel-preset-gatsby",
+        {
+          "targets": {
+            "browsers": [">0.25%", "not dead"],
+          },
+        },
+      ],
+    ],
+    "plugins": ["@babel/plugin-proposal-class-properties"]
```

Comment:
Is there a plugin we're using that requires this? If it's just our code, I'd rather we don't and just stick to ES6

Reply:
Done